### DISCUSSION TITLE
Kong IP Restriction plugin returns "Kong Error" in error message

### DISCUSSION THREAD

**Discussion by majid-akbari** [Kong/kong#11669] [Ideas and feature requests]
Is it possible to remove this `Kong Error` response from Kong IP Restriction plugin error message, it helps to hide topology from attackers.

**Comment by scrudge** [comment]
Actually this would apply to all kong.response.error calls that have [html mime](https://github.com/Kong/kong/blob/master/kong/tools/utils.lua#L698) type.  This would be a great feature. +1
